catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/taxonomy/MetadataAttributes.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.impl.taxonomy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.taxonomy.Metadata;
+
+public class MetadataAttributes implements Metadata, MetacardType {
+
+    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+
+    private static final String NAME = "metadata-metacard";
```

Comment:
Not sure that anyone is going to understand `provenance`.  My recommendation is to move these to the metacard category.  I think we want created/modified attributes for the metacard itself (rather than the "original" metadata.